scripts/crontab/gen-cron.py

```diff
@@ -26,13 +26,8 @@ def main():
     if not opts.deprecations:
         opts.python += ' -W ignore::DeprecationWarning'
 
-    dogwrap_path = '/usr/local/bin/amo_cron_dogwrap'
+    ctx = { "django": "cd %s; %s %s manage.py" % (opts.zamboni, opts.python) }
```

Comment:
This doesn't appear valid python - the string still has 3 substitutions but now only 2 values have been provided

Reply:
Oops, I thought I removed that. 😝 

Thank you for catching this!